### PR TITLE
SIMD-0138: Deprecate legacy vote instructions

### PR DESCRIPTION
**Pull request by AshwinSekar**

### CODE REVIEW

File: proposals/0138-deprecate-legacy-vote-instructions.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Impact
+
+Sending transactions that include the mentioned instructions will fail.
+
+## Security Considerations
+
+Implementations should ensure that the `TowerSync` instruction has been
+activated before enabling the `deprecate legacy vote instructions` feature,
+in order to ensure that at least one vote instruction available for the client.
```
Comment (topointon-jump) [inline proposals/0138-deprecate-legacy-vote-instructions.md:52]: Do you mind specifying which feature gate the `TowerSync` instruction is gated by?

Comment (topointon-jump) [inline proposals/0138-deprecate-legacy-vote-instructions.md:52]: Minor nit: this isn't really down to the implementation, as feature activations are cluster-wide.

If you do mean that the implementation should explicitly do this check, could you specify that? If not, could you re-word to make clear that this is the responsibility of the feature activator? 

Thanks! 🙇

Comment (AshwinSekar) [reply proposals/0138-deprecate-legacy-vote-instructions.md]: updated to be clear on the interaction of these 2 features

**Review by carllin — APPROVED** [review]

**Review by topointon-jump — APPROVED** [review]

**Comment by jacobcreech** [comment]
> We have consensus on the Anza & Firedancer side. Suggest we give this SIMD 1 week time for community comments and then merge.

Alright, we'll merge on April 18th, 2024 if no other comments against this come up.

**Comment by lheeger-jump** [comment]
Merging, no new comments.

**Pull request merged**